### ISSUE TITLE
Image TAB is missing

### ISSUE THREAD

**Issue by rafaelmr2** [specify/webportal-installer#9]
Hi,
I'm using Ubuntu 16.04.3 updated with Tomcat 7. Images TAB is missing (see image attached).
![specify_imgtab_missing](https://user-images.githubusercontent.com/4471805/36389304-69a0fa88-157d-11e8-9be3-c0660d4cf439.png)

java version "1.8.0_161"

Thanks,

**Comment by timo11** [comment]
If the image tab is not there at all it may because the imageBaseUrl  setting for the image server, in resources/config/settings.json, is missing. (e.g. "imageBaseUrl": "http://biimages.biodiversity.ku.edu")

If images are not showing due to bad request errors it could be related to the specific version of Tomcat you are using. https://github.com/specify/webportal-installer/issues/7#issuecomment-366670146

**Comment by rafaelmr2** [comment]
Doesn't work. I have to unzip the file and change "imageBaseUrl" in file PortalFiles/PortalInstanceSetting.json . It works.

**Comment by rafaelmr2** [comment]
Doesn't work. I have to unzip the file and change "imageBaseUrl" in file PortalFiles/PortalInstanceSetting.json . It works.

**Comment by misilot** [comment]
Yeah it doesn't appear you can use custom_settings either to override or set `imagebaseurl`
